frontend/www/js/omegaup/components/arena/Contest.vue

```diff
@@ -0,0 +1,257 @@
+        @clarification-response="
+          (id, responseText, isPublic) =>
+            $emit('clarification-response', id, responseText, isPublic)
+        "
+        @update:activeTab="
+          (selectedTab) => $emit('update:activeTab', selectedTab)
+        "
+      ></omegaup-arena-clarification-list>
+    </template>
+  </omegaup-arena>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
+import { types } from '../../api_types';
+import * as ui from '../../ui';
+import T from '../../lang';
+import arena_Arena from './Arena.vue';
+import arena_ClarificationList from './ClarificationList.vue';
+import arena_NavbarProblems from './NavbarProblems.vue';
+import arena_NavbarMiniranking from './NavbarMiniranking.vue';
+import arena_ContestSummary from './ContestSummaryV2.vue';
+import arena_Scoreboard from './Scoreboard.vue';
+import omegaup_Markdown from '../Markdown.vue';
+import problem_Details, { PopupDisplayed } from '../problem/Details.vue';
+import { omegaup } from '../../omegaup';
+
+export interface ActiveProblem {
+  runs: types.Run[];
+  problem: types.NavbarProblemsetProblem;
+}
+
+@Component({
+  components: {
+    'omegaup-arena-clarification-list': arena_ClarificationList,
+    'omegaup-arena': arena_Arena,
+    'omegaup-arena-contest-summary': arena_ContestSummary,
+    'omegaup-arena-navbar-miniranking': arena_NavbarMiniranking,
+    'omegaup-arena-navbar-problems': arena_NavbarProblems,
+    'omegaup-arena-scoreboard': arena_Scoreboard,
+    'omegaup-markdown': omegaup_Markdown,
+    'omegaup-problem-details': problem_Details,
+  },
+})
+export default class ArenaContest extends Vue {
+  @Prop() contest!: types.ContestPublicDetails;
+  @Prop() contestAdmin!: boolean;
+  @Prop() problems!: types.NavbarProblemsetProblem[];
+  @Prop({ default: () => [] }) users!: types.ContestUser[];
+  @Prop({ default: null }) problem!: ActiveProblem | null;
+  @Prop() problemInfo!: types.ProblemInfo;
+  @Prop({ default: () => [] }) clarifications!: types.Clarification[];
+  @Prop({ default: false }) isEphemeralExperimentEnabled!: boolean;
+  @Prop({ default: false }) admin!: boolean;
+  @Prop({ default: true }) showNavigation!: boolean;
+  @Prop({ default: false }) showRanking!: boolean;
+  @Prop({ default: true }) showClarifications!: boolean;
+  @Prop({ default: true }) showDeadlines!: boolean;
+  @Prop({ default: false }) isAdmin!: boolean;
+  @Prop({ default: false }) showNewClarificationPopup!: boolean;
+  @Prop({ default: PopupDisplayed.None }) popupDisplayed!: PopupDisplayed;
+  @Prop() activeTab!: string;
+  @Prop({ default: null }) guid!: null | string;
+  @Prop() minirankingUsers!: omegaup.UserRank[];
+  @Prop() ranking!: types.ScoreboardRankingEntry[];
+  @Prop() scoreboardColors!: string[];
+  @Prop() lastUpdated!: Date;
+  @Prop() digitsAfterDecimalPoint!: number;
+  @Prop() showPenalty!: boolean;
+  @Prop({ default: true }) socketStatus!: boolean;
```

Comment:
```suggestion
  @Prop({ default: true }) socketConnected!: boolean;
```

"status" no suena muy booleanoso.

Reply:
Listo. 

Noté que hay un tercer estado del socket (que se representa con ↻), ahorita lo dejaré así, pero después tendremos que revisar cual es el nombre que mejor describa a esta prop